### PR TITLE
Crypt of the NecroDancer

### PR DESCRIPTION
**Pull request by espeon65536**
So, ND is in a little bit of a weird place. The mod works in the game, but there isn't "official" support for Lua modding yet even though the system is fully functional behind the scenes. However we're not expecting that support to be official anytime soon, so we may as well get it running since the setup will only get easier later on. 

The Lua mod can be found [here](https://github.com/espeon65536/APNecrodancer/releases) and should probably be included in the release notes once this is merged. The tutorial links to it as well. 

I'm not marking this as draft because it is ready to be merged, but I'm not expecting this to make it into 0.3.4 at all; it's way too close to the feature freeze and it hasn't been tested remotely close to enough.

### CODE REVIEW

**Comment by Jarno458** [comment]
Had a look at the code, seemed decent enough

File: worlds/cotnd/docs/setup_en.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### Where do I get a config file?
+
+The Player Settings page on the website allows you to configure your personal settings and export a config file from
+them. Player settings page: [Crypt of the NecroDancer Player Settings Page](/games/Crypt%20of%20the%20NecroDancer/player-settings)
+
+**Important**: The option `available_characters` will likely be missing from your YAML after downloading. This option is quite important
+as it allows you to control the characters available for gameplay. You can add it like this:
+```yaml
+available_characters: [Cadence, Melody, Aria, Nocturna]
```
Comment (SoldierofOrder) [inline worlds/cotnd/docs/setup_en.md:65]: Could you make it a little clearer for utter noobs what they need to do to include this line? Assume they've never heard of the .yaml file extension before.

**Review by SoldierofOrder — CHANGES_REQUESTED** [review]
I only examined the docs. Looks great! Just a handful of things I had suggestions for. :)

**Comment by espeon65536** [comment]
Based on people's feedback and my own opinion, the Lua mod for the game is far too underdeveloped. I have some ideas for improvement, but this will take a significantly longer time than anticipated and will involve significant changes to the AP side (once I prove that the ideas can be done), probably long enough that this is not worth keeping open.

**Pull request closed without merging**